include/barrett/bus/abstract/communications_bus.h

```diff
@@ -45,7 +45,7 @@ namespace bus {
 class CommunicationsBus {
 public:
 	static const size_t MAX_MESSAGE_LEN = 8;  /** The maximum of any of the available communications buses */
-	static const double TIMEOUT = 1.0;  /** Bus connection timeout limit in seconds */
+	static const double TIMEOUT = 0.01;  /** Bus connection timeout limit in seconds */
```

Comment:
This is actually used every time when waiting for response can frames (it receives, sleeps, loops until total passed time is greater than this timeout whenever it's waiting for a response frame). I tested late last night and it seems like it's not necessary, although 1 second is absurd for this, because we're way past the ability to recover from a missed message. 10 ms is potentially recoverable in software (or at least can exit more gracefully).